instrumentation/netty/netty-4-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/common/AbstractNettyChannelPipelineInstrumentation.java

```diff
@@ -48,10 +49,17 @@ public void transform(TypeTransformer transformer) {
         isMethod().and(named("remove").or(named("replace"))).and(takesArgument(0, Class.class)),
         AbstractNettyChannelPipelineInstrumentation.class.getName() + "$RemoveByClassAdvice");
     transformer.applyAdviceToMethod(
-        isMethod()
-            .and(named("removeFirst").or(named("removeLast")))
-            .and(returns(named("io.netty.channel.ChannelHandler"))),
-        AbstractNettyChannelPipelineInstrumentation.class.getName() + "$RemoveFirstLastAdvice");
+        isMethod().and(named("removeFirst")).and(returns(named("io.netty.channel.ChannelHandler"))),
+        AbstractNettyChannelPipelineInstrumentation.class.getName() + "$RemoveFirstAdvice");
+    transformer.applyAdviceToMethod(
+        isMethod().and(named("removeLast")).and(returns(named("io.netty.channel.ChannelHandler"))),
+        AbstractNettyChannelPipelineInstrumentation.class.getName() + "$RemoveLastAdvice");
+    transformer.applyAdviceToMethod(
+        isMethod().and(named("addAfter")).and(takesArguments(3)),
```

Comment:
The 3-args variant calls the 4-args one, it should be enough to instrument the 4-args one.

Reply:
ok i got it